assets/Icons/README.md

```diff
@@ -1,44 +1,17 @@
 # Source SVG icons used to build React components
 
-### Here we collect the raw (exported from Sketch) and optimized SVGs.
+## To add a new icon or icons
 
-1. Add raw SVGs to assets/Icons/raw folder
+1. Add prepared SVGs to assets/Icons/optimized folder
 
-2. Process SVGs with svgo
+2. Run `yarn create:svg:components`
```

Comment:
can we use a simpler command like `yarn svg`?

Reply:
Sounds good to me.